app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/mysong/subfragment/listsong/ListSongFragment.java

```diff
@@ -3,18 +3,28 @@
 import android.os.Bundle;
 import android.support.annotation.NonNull;
 import android.support.v4.app.Fragment;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports